astropy/table/tests/test_table.py

```diff
@@ -1052,10 +1072,24 @@ def test_single_big(self, table_types):
         assert np.all(t['x'] == x[idx])
         assert np.all(t['y'] == y[idx])
 
+    def test_single_big_reverse(self, table_types):
```

Comment:
I wonder if we could/should piggy back this on `test_single_big` to avoid creating `x = np.arange(10000)` twice, or maybe it is premature optimization. Just thinking out loud here.

Reply:
Thank you @pllim for having a look at my PR. I agree to your point. I could do it if you say so. For now, I'm pushing again after fixing other errors.